components/aws/sagemaker/TrainingJob/README.md

```diff
@@ -0,0 +1,99 @@
+# SageMaker Training Kubeflow Pipelines component
```

Comment:
Has this folder structure been discussed already ? having both train and TrainingJob seems confusing.

Reply:
fair point. Added Versioning in the main README and changed the title to include v2 for now. Maybe we can use the same mechanism for other components as well